src/asmcnc/apps/start_up_sequence/welcome_to_smartbench_app/screens/screen_language_select.py

```diff
@@ -379,7 +379,11 @@ def change_welcome_label(self, dt):
 		if self.welcome_i < 6:
 			self.welcome_i += 1
 
+		elif self.welcome_i == 7:
```

Comment:
How is this condition triggered? Wouldn't it go to the else condition when `self.welcome_i == 6` and get set back to 0?